src/controllers/bmdashboard/bmNewLessonController.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const bmNewLessonController = function (BuildingNewLesson) {
+    const bmGetLessonList = async (req, res) => {
+        try {
+            BuildingNewLesson
+            .find()
+            .populate()
+            .then(result => res.status(200).send(result))
+            .catch(error => res.status(500).send(error));
+        } catch (err) {
+            res.json(err);
+        }
+    };
+    const bmPostLessonList = async (req, res) => {
+        try {
+            const { id, title, content, author, tag, relatedProject} = req.body;
```

Comment:
- You don't need to deconstruct `req.body` if you are not using the variables. Line 17 can either be omitted or used to define a new variable such as `const newDoc = req.body; const newLesson = BuildingNewLesson.create(newDoc);`
- Mongo ignores fields that don't match the model. `tag` should be `tags`.